### ISSUE TITLE
[INSTALL]: 

### ISSUE THREAD

**Issue by JohnDamilakis** [pycaret/pycaret#3413]
### Installation check

- [X] I have read the [installation guide](https://pycaret.gitbook.io/docs/get-started/installation).

### Platform

windows 10 64bit

### Installation Method

pip install

### pycaret Version

All versions 

### Python Version

N/A, I'm using colab

### Description

ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.
pandas-profiling 3.2.0 requires joblib~=1.1.0, but you have joblib 1.2.0 which is incompatible.

### Installation Logs

<details>

Replace this line with the installation logs.

</details>

**Comment by ngupta23** [comment]
I saw the same, but it did not impact the working. Did you have any issues running the code?

**Comment by ngupta23** [comment]
By the way, Colab uses linux not windows.

**Comment by moezali1** [comment]
@JohnDamilakis I can't reproduce this error with `pip install pycaret` on Colab. Is it coming in `pycaret[full]` install?

**Comment by JohnDamilakis** [comment]
@ngupta23 Yes, I have issues running the code. The next Google colab cell 
from pycaret.utils import enable_colab
enable_colab()
doesn't work.

**Comment by ngupta23** [comment]
What is the exact error message. Please provide it. I dont think `enable_colab` is needed anymore.
